### PR TITLE
Completed-Css-Layout-Classes.

### PR DESCRIPTION
**Pull request by Mitchelllund**
Screenshot 2022-05-23 205427css-layout-classes

### CODE REVIEW

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        <img src="images/yoshi.png" alt="Yoshi">
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-full">
+        <img src="images/kirby.png" alt="Kirby">
+        <img src="images/fox.png" alt="Fox">
+        <img src="images/pikachu.png" alt="Pikachu">
+        <img src="images/ness.png" alt="Ness">
+      </div>
```
Comment (BrettAlbright) [inline css-layout-classes/index.html]: you want to have four separate columns here, one for each image.

File: css-layout-classes/styles.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  max-width: 800px;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
```
Comment (BrettAlbright) [inline css-layout-classes/styles.css]: you don't need to set the flex-direction if you are using the default size

File: css-layout-classes/styles.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  margin-bottom: 8px;
+}
+
+.column-one-third {
+  display: flex;
+  flex-direction: column;
```
Comment (BrettAlbright) [inline css-layout-classes/styles.css]: you don't need to alter the flex direction for this exercise.

File: css-layout-classes/styles.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
+.container {
+  max-width: 800px;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```
Comment (BrettAlbright) [inline css-layout-classes/styles.css]: I would avoid using display flex on the container itself. That is typically for the rows.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
overall solid, a few updates are needed.

File: css-layout-classes/styles.css (hunk 4 above)
Comment (Mitchelllund) [reply css-layout-classes/styles.css]: Awesome! Thanks for the feedback!

**Comment by Mitchelllund** [comment]
Screenshot 2022-05-23 205427css-layout-classes

File: css-layout-classes/styles.css [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  margin-bottom: 8px;
+}
+
+.column-one-third {
+  width: calc(100% / 3);
+}
+
+img {
+  width: 100%;
+  height: 100%;
```
Comment (BrettAlbright) [inline css-layout-classes/styles.css]: I would remove the height here, you just want to set either width or height for images, and in this case, width is what you want. That will allow the image to automatically maintain the correct aspect ratio.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
one tiny update!

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**